pip-selfcheck.json

```diff
@@ -0,0 +1 @@
+{"last_check":"2016-08-06T03:27:25Z","pypi_version":"8.1.2"}
```

Comment:
Can you remove this file?

And then add pip-selfcheck.json to the bottom of .gitignore so git ignores it in the future